### DISCUSSION TITLE
about http acme

### DISCUSSION THREAD

**Discussion by ryan053857** [roadrunner-server/roadrunner#2060] [Show and tell]
I tried https amce, and I found that when I failed to get the certificate, I would keep trying, blocking the actual http process, can't this be changed to asynchronous?

**Comment by rustatian** [comment]
How does RR continue to serve HTTP**S** traffic when it failed to get certs?

**Reply by ryan053857** [reply]
But the http service is also unavailable when  it failed to get certs
